enterprise_access/apps/events/apps.py

```diff
@@ -0,0 +1,31 @@
+"""
+Initialization app for enterprise_access.apps.events.
+"""
+
+import logging
+
+from django.apps import AppConfig
+from django.conf import settings
+from edx_toggles.toggles import SettingToggle
+
+from enterprise_access.apps.events.utils import create_topics
+
+logger = logging.getLogger(__name__)
+
+KAFKA_ENABLED_TOGGLE = SettingToggle("KAFKA_ENABLED", default=False)
```

Comment:
do we still want this or are we just going to use settings.KAFKA_ENABLED?

Reply:
yeah we don't need this, will remove